homeassistant/components/speedtestdotnet/__init__.py

```diff
@@ -32,6 +32,8 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+SENSOR_KEYS = [desc.key for desc in SENSOR_TYPES]
+
 CONFIG_SCHEMA = vol.Schema(
     vol.All(
         # Deprecated in Home Assistant 2021.6
```

Comment:
Is this comment not correct?

Reply:
That comment is correct (#50072), but it was not removed to 2021.8, and to 2021.9 `SENSOR_KEYS` changed, and it's now causing issues for those who still have not removed it from their configuration.